### PR TITLE
Fixes #22741: Rewrite angular app "Node properties" in Elm

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://issues.rudder.io/issues/22741

This is part of the plan to completely remove AngularJs from Rudder, for reasons of security and code maintenance.

No new features have been added, this is a rewrite of the AngularJs "nodeProperties" application in Elm. 
The js files related to the old application have been removed

### CODE REVIEW

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
PR rebased

**Comment by RaphaelGauthier** [comment]
PR updated with a new commit

**Comment by RaphaelGauthier** [comment]
PR rebased

**Comment by RaphaelGauthier** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/elm/sources/NodeProperties/JsonEncoder.elm [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    encodeProp p =
+      let
+        value = if action == "Delete" then string "" else
+          case p.format of
+            JsonFormat ->
+              decodeString Json.Decode.value p.value
+                |> Result.withDefault (Json.Encode.string p.value)
+            StringFormat -> string p.value
+      in
+      D.log ("=================")
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/elm/sources/NodeProperties/JsonEncoder.elm:24]: Debug log here

**Comment by RaphaelGauthier** [comment]
PR updated with a new commit

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**